function_programming_purrr.Rmd

```diff
@@ -0,0 +1,150 @@
+# Functional Programming with Purrr
+
+Ran Pan
+
+```{r setup, include=FALSE}
```

Comment:
Please modify this. The instruction is the following:
If the file contains a setup chunk in .Rmd file, it should not contain a setup label. (The bookdown render will fail if there are duplicate chunk labels.)
i.e. use {r, include=FALSE} instead of {r setup, include=FALSE}.